xdsl/interpreters/riscv_emulator.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import annotations
+from typing import IO, ClassVar
+
+from riscemu.config import RunConfig
+from riscemu import RV32I, RV32M
+from riscemu.CPU import UserModeCPU
+from riscemu.parser import AssemblyFileLoader
+from riscemu.instructions.instruction_set import InstructionSet
+from riscemu.types.instruction import Instruction
```

Comment:
This should probably be in a `try .. catch ...` block somewhere instead of the top level?